src/main/res/values/do_not_translate.xml

```diff
@@ -99,4 +99,32 @@ limitations under the License.
         <item>@string/waypoint_type_water_fountain</item>
         <item>@string/waypoint_type_waypoint</item>
     </string-array>
+
+    <string-array name="show_on_map_format_labels">
+        <item>@string/show_on_dashboard</item>
+        <item>@string/show_on_map_as_KMZ_with_photos</item>
+        <item>@string/show_on_map_as_KML</item>
+        <item>@string/show_on_map_as_GPX</item>
+    </string-array>
+    <string-array name="show_on_map_format_values" translatable="false">
+        <item>Intent.OpenTracks-Dashboard</item>
+        <item>KMZ_WITH_TRACKDETAIL_AND_SENSORDATA_AND_PICTURES</item>
+        <item>KML_WITH_TRACKDETAIL_AND_SENSORDATA</item>
+        <item>GPX</item>
+    </string-array>
+    <string-array name="show_on_map_format_settings_labels">
+        <item>@string/show_on_map_format_ask</item>
+        <item>@string/show_on_dashboard</item>
+        <item>@string/show_on_map_as_KMZ_with_photos</item>
+        <item>@string/show_on_map_as_KML</item>
+        <item>@string/show_on_map_as_GPX</item>
+    </string-array>
+    <string-array name="show_on_map_format_settings_values" translatable="false">
+        <item>@string/show_on_map_format_ask_value</item>
+        <item>Intent.OpenTracks-Dashboard</item>
```

Comment:
Then I'll go for `OpenTracks Dashboard API`, because the OSM Dashboard might not be the only one. Do we need to mention, that an extra installation might be necessary?

Reply:
When using it, there would be an error message if no suitable application is installed, right?
I guess that is okay for now ;)

Reply:
Right 👍